dc/s2n-quic-dc/src/stream/environment/udp.rs

```diff
@@ -83,15 +83,51 @@ where
 
     #[inline]
     fn setup(self, _env: &E) -> SetupResult<Self::ReadWorkerSocket, Self::WriteWorkerSocket> {
-        let remote_addr = self.peer_addr;
+        let mut remote_addr = self.peer_addr;
         let control = self.control;
         let stream = self.stream;
         let queue_id = control.queue_id();
 
+        let local_addr: SocketAddress = self.worker_socket.local_addr()?.into();
         let application = Box::new(self.application_socket);
         let read_worker = Some(self.worker_socket.clone());
         let write_worker = Some((self.worker_socket, buffer::Channel::new(control)));
 
+        fn ipv6_loopback() -> IpV6Address {
+            let mut octets = [0; 16];
+            octets[15] = 1;
+            IpV6Address::new(octets)
```

Comment:
Maybe ::from(https://doc.rust-lang.org/nightly/std/net/struct.Ipv6Addr.html#associatedconstant.LOCALHOST)?

Reply:
Yeah I can change it